packages/app-page-builder/src/blockEditor/config/eventActions/saveBlock/getPreviewImage.ts

```diff
@@ -23,10 +24,14 @@ function getDataURLImageDimensions(dataURL: string): Promise<ImageDimensionsType
     });
 }
 
-interface createdImageType {
-    data: any;
+interface CreatedImageType {
```

Comment:
I missed this part in my reviews earlier; it would be a lot better if `getPreviewImage` returned `null` if there's no plugin, rather than an empty object. Ideally, the return type of `getPreviewImage` is `Promise<File | null>`.